### PR TITLE
feat: integrated dynamic breadcrumbs into PageHeader

### PR DESCRIPTION
**Pull request by jeffski13**
I modified the Pageheader to use the new dynamic Breadcrumbs.
Props validation added for crumbs and custom children BreadcrumbItems

### CODE REVIEW

File: packages/breadcrumbs/Breadcrumbs.js [hunk 1]
```diff
@@ -49,16 +49,40 @@ Breadcrumbs.propTypes = {
...
-
+    if (!props.children) {
+      return null;
+    }
     let error = null;
-    React.Children.forEach(prop, child => {
-      if (child.type !== BreadcrumbItem) {
+
+    // multi children
+    if (Array.isArray(props.children)) {
```
Comment (robmcguinness) [inline packages/breadcrumbs/Breadcrumbs.js:58]: Can we add a corresponding test case using `react-testing-library`?

Comment (jeffski13) [reply packages/breadcrumbs/Breadcrumbs.js:58]: DONE!

**Comment by jeffski13** [comment]
All i did was update some new unit tests, and now Travis hates me ):

**Comment by GoPro16** [comment]
@robmcguinness  Looks like the build is starting to take longer now since we are adding more components to the story book.  Can you try re-starting the netlify build? I am pretty sure this is the case that is happening right now.

**Comment by GoPro16** [comment]
@jeffski13  review the changes I just made and let me know if you see any issues with it.

**Pull request merged**